x-pack/plugins/canvas/public/components/workpad_templates/templates/index.js

```diff
@@ -0,0 +1,10 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+const themeDark = require('./theme_dark.json');
+const themeLight = require('./theme_light.json');
+
+export const templates = [themeDark, themeLight];
```

Comment:
Make this business a registry like we have for elements, functions, etc. That'll set us up for pluggable dashboards right quick.